test/instances/default.lua

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env tarantool
```

Comment:
Somewhy I thought it works in the same way even for 1.10...

You can also add this issue to commit message - https://github.com/tarantool/tarantool/issues/3167

Reply:
Done.